store/src/aggregations.rs

```diff
@@ -233,14 +233,18 @@ impl Agregation for AgregationStoreGoods {
         self.receive.cost += cost;
       },
       InternalOperation::Issue(qty, cost, mode) => {
-        log::debug!("fn_apply_operation: self.issue {:?}\nqty {:?}", self.issue, qty);
         self.issue.qty -= qty;
-        // TODO 0.0001 left if 10/3
         if mode == &Mode::Auto {
           let balance = self.open_balance.clone() + self.receive.clone();
-          // let cost = balance.price(name.clone()).cost(qty.clone(), name);
-          let cost = qty.cost(&balance);
-          log::debug!("fn_apply_operation: cost {cost:?}\nqty {:?}", self.issue.qty);
+          let cost = match self.issue.qty.abs().is_greater_or_equal(&balance.qty) {
```

Comment:
this is wrong!

Reply:
For example we have:
qty: 3, cost: 10
two issue with Mode::Auto:
-1 cost: 3.3333
-2 cost: 6.6666
And we have 0.0001 cost left. How to handle it in another way?

Reply:
when qty eq balance then all cost must be used with out price calculation